app/views/prison/visits/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :header do %>
+  <%= t('.title') %>
+<% end %>
+
+<%= render(partial: 'people_box') %>
+
+<% unless @booking_response.processable? %>
+  <%= reason_reqeuest_cannot_be_processed(@booking_response) %>
+<% else %>
+
+<div class="processing-form confirmations">
+  <%= t('.introduction_html') %>
+
+  <%=
+    form_for(
+      @booking_response,
+      url: prison_visit_path(@booking_response),
+      method: :put
+    ) do |f|
+  %>
+<%= render(partial: 'visit_date_section', locals: { f: f }) %>
+<%= render(partial: 'visiting_allowance_section', locals: { f: f }) %>
+<%= render(partial: 'issue_with_prisoner_section', locals: { f: f }) %>
+<%= render(partial: 'issue_with_visitors_section', locals: { f: f }) %>
```

Comment:
The indentation is off. Luckily, we're not using Haml, so we don't get a totally different HTML document, but it would be nice to indent them properly nonetheless!